service-python/featuretoggle/src/requirements.txt

```diff
@@ -0,0 +1,4 @@
+cerberus==1.3.4;
+python-dateutil==2.8.2;
+pika==1.3.1
+redis==4.3.4
```

Comment:
Why do some lines end in `;`?

Reply:
no idea, i'll clean it up. i copied this from the requirements.txt in the `assessclaimdc7101` where it is similarly mismatched :)